cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -568,6 +571,27 @@ public static Admin adminClient()   {
         }
         when(mock.describeMetadataQuorum()).thenReturn(dmqr);
 
+        DescribeClientQuotasResult dcqr;
+        try {
+            Constructor<DescribeClientQuotasResult> declaredConstructor = DescribeClientQuotasResult.class.getDeclaredConstructor(KafkaFuture.class);
+
+            Map<String, Double> result = new HashMap<>();
+            result.put("producer_byte_rate", 20000.0);
+            result.put("consumer_byte_rate", 20000.0);
+            result.put("request_percentage", null);
+            result.put("controller_mutation_rate", null);
+
+            KafkaFuture<Map<ClientQuotaEntity, Map<String, Double>>> future = KafkaFuture.completedFuture(Map.of(QuotasPluginUtils.DEFAULT_USER_ENTITY, result));
+
+            declaredConstructor.setAccessible(true);
+            dcqr = declaredConstructor.newInstance(future);
+        } catch (NoSuchMethodException | InstantiationException | IllegalAccessException |
+                 InvocationTargetException e) {
+            throw new RuntimeException(e);
+        }
+
+        when(mock.describeClientQuotas(any())).thenReturn(dcqr);
+
```

Comment:
This seems weird. Why do you mock it for every test using this to return some quotas?

Reply:
In the tests when there is usage of `reconcile`, it tries to describe the quotas using the admin client, which ends up in exception. That's why I added it to the `ResourceUtils` for all the tests.

Reply:
Right, I understand that. But normally the mock would be to return it empty as the default quotas will not be set in most tests. This way, if your code works, in most tests this would also trigger an update to delete the quota which is something you do not want in unrelated tests.

Reply:
That's true, I'm gonna return empty quotas as default. Thanks :)